_data/tools.yml

```diff
@@ -2191,3 +2191,13 @@
   link: https://github.com/criteo/openapi-comparator
   description: C# library for comparing two OpenAPI specifications.
   v3: true
+
+- name: Specmatic
+  category:
+    - mock
+    - testing
+  language: Any, CLI
+  github: https://github.com/znsio/specmatic
+  link: https://specmatic.in/
+  description: Specmatic enables teams to practice Contract-Driven Development by leveraging OpenAPI specifications as Executable Contracts to reduce time-to-market by allowing teams to build, test and deploy their MicroServices and MicroFrontends independently and Shift-Left the identification of compatibility issues between Components / Services in complex interdependent systems such as those built with MicroServices architecture. In the process it reduces / removes the need for Integration Testing.
+  v3: true
```

Comment:
Please include a line break after the entry.

Reply:
@daveshanley thanks for taking a look at the PR. I have added the line break.